spec/controllers/selects_collections_controller_spec.rb

```diff
@@ -9,8 +9,9 @@ class SelectsCollectionsController < ApplicationController
 describe SelectsCollectionsController, type: :controller do
   describe "#find_collections" do
     it "uses the search builder" do
-      expect(subject.collections_search_builder_class.default_processor_chain).to eq [:default_solr_parameters, :add_query_to_solr, :add_access_controls_to_solr_params, :add_collection_filter, :some_rows, :sort_by_title]
-      expect(CurationConcerns::CollectionSearchBuilder).to receive(:new).with(subject).and_call_original
+      expect(subject.collections_search_builder_class.default_processor_chain)
+        .to include(:default_solr_parameters, :add_query_to_solr, :add_access_controls_to_solr_params)
```

Comment:
I do that in `controller_search_builder_spec.rb`.  The class was not tested previously, but now that it is, several tests are more appropriate there.